VideoWeb/VideoWeb/AuthenticationSchemes/EJudiciaryScheme.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+
+namespace VideoWeb.AuthenticationSchemes
+{
+    public class EJudiciaryScheme : ProviderSchemeBase, IProviderSchemes
+    {
+        public EJudiciaryScheme(string eventhubPath): base(eventhubPath)
+        {
+        }
+
+        public override AuthProvider Provider => AuthProvider.EJudiciary;
+
+        public bool BelongsToScheme(JwtSecurityToken jwtSecurityToken) => jwtSecurityToken.Issuer.Contains("0b90379d-18de-426a-ae94-7f62441231e0", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
would it be safer to check the issuer from a config?

Reply:
yes :P todo